src/main/java/seedu/address/model/book/Rating.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.model.book;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a book's rating.
+ * Guarantees: immutable.
+ */
+public class Rating {
+
+    public final Integer rating;
+
+    /**
+     * Constructs a {@code Rating}.
+     *
+     * @param rating A book rating.
+     */
+    public Rating(Integer rating) {
```

Comment:
Use an `int` instead?

Reply:
Using int will cause some error in the following override methods, so I changed to Integer